en/DatabaseProperties.md

```diff
@@ -65,3 +65,11 @@ If you check **Enable save actions**, the list of actions can be configured.
 Each action is defined by:
 - an entry field (upon which the action will be applied).
 - the type of action to be carried out (such as *HTML to LaTeX*, which converts HTML code to LaTeX code, as described in the window).
+
+## Shared database
+
```

Comment:
"Note" instead of "Consider"? Consider works, so no need to change it, but to me (also non-native though) it sounds slightly worse than e.g. note.